src/main/java/uk/nhs/tis/sync/event/listener/JobRunningListener.java

```diff
@@ -48,13 +48,17 @@ public class JobRunningListener implements ApplicationListener<ApplicationReadyE
   @Autowired
   private PersonElasticSearchSyncJob personElasticSearchSyncJob;
 
-  @Autowired
   private RevalCurrentPmSyncJob revalCurrentPmSyncJob;
 
   private LocalTime earliest;
 
   private LocalTime latest;
 
+  @Autowired(required = false)
+  public void setRevalCurrentPmSyncJob(RevalCurrentPmSyncJob revalCurrentPmSyncJob) {
+    this.revalCurrentPmSyncJob = revalCurrentPmSyncJob;
+  }
+
```

Comment:
All the other jobs use annotation on the fields.  Perhaps in a later PR, would you think about whether it makes more sense to use field injection or constructor+setter injection?